DEVELOPERS.rst

```diff
@@ -0,0 +1,68 @@
+For developers of ZODB
+======================
```

Comment:
Would you like to use the "standard" RST headers now that this is its own document? E.g., line above and below the document title? If you're in Emacs, C-c C-a C-s does that for you.